### PR TITLE
add nvpsmpi toolchain definition (NVHPC + ParaStationMPI)

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — APPROVED** [review]
lgtm

File: easybuild/toolchains/nvpsmpi.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for npsmpi compiler toolchain (includes NVHPC and ParaStationMPI).
+
+:author: Damian Alvarez (Forschungszentrum Juelich)
+:author: Sebastian Achilles (Forschungszentrum Juelich)
```
Comment (boegel) [inline easybuild/toolchains/nvpsmpi.py:30]: @robert-mijakovic This should be changed to mention yourself?

File: easybuild/toolchains/nvpsmpi.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for npsmpi compiler toolchain (includes NVHPC and ParaStationMPI).
```
Comment (boegel) [inline easybuild/toolchains/nvpsmpi.py:27]: `npsmpi` should be `nvpsmpi`?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by ocaisa** [comment]
@robert-mijakovic This looks good to merge, just need to address the comments

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**